src/compiler/vdom-parser.js

```diff
@@ -28,7 +28,7 @@ function extractDirectives(node) {
       attrs.push({ name, value })
     }
     else if(name.startsWith('@')) {
-      name = `on${name.slice(1)}`
+      if (!name.includes('-')) name = `on${name.slice(1)}`
```

Comment:
If custom events are turned into `oncustom-event`, they finally arrive in Lego component as `oncustomevent` (`petit-dom` magic?), making it impossible to covnert in camelCase.

Reply:
I'm not sure I get the use case.
Right now it would behave as `@custom-event` -> `oncustom-event`, which is not gread indeed.
But in which case would we do `@custom-event`? What would be the proper conversion and meaning of `@` in that case?